tools/src/main/scala/scala/scalanative/build/NativeConfig.scala

```diff
@@ -57,6 +60,9 @@ sealed trait NativeConfig {
   /** Create a new config with given compilation options. */
   def withCompileOptions(value: Seq[String]): NativeConfig
 
+  /** Create a new config given a target triple. */
+  def withTargetTriple(value: Option[String]): NativeConfig
```

Comment:
Consider adding an overload that takes a `String` and delegates to this one with `Some(value)`. That will be easier to use in most configs.

Reply:
Good idea.